ci/code_checks.sh

```diff
@@ -181,6 +181,7 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         pandas.core.groupby.SeriesGroupBy.rolling\
         pandas.core.groupby.DataFrameGroupBy.hist\
         pandas.core.groupby.DataFrameGroupBy.plot\
+        pandas.core.groupby.DataFrameGroupBy.corrwith\
```

Comment:
I was going to add groupby-specific documentation to this method (rather than just repeating the DataFrame version), but the more I look at it, the more I think we should just deprecate this method. Will make an issue.

Reply:
Opened #57158